core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/rabbitmq.yml

```diff
@@ -8,7 +8,7 @@
   register: rabbitmq_version_current
   changed_when: false
 
-- name: RabbitMQ | Include defaults from node_exporter role
+- name: RabbitMQ | Include defaults from rabbitmq role
```

Comment:
This is a typo from previous PR that has been already merged, imo we don't need a separate PR for that.